<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/spacebg.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.175325 rgba(170, 85, 127, 255), stop:0.321429 rgba(122, 122, 255, 255), stop:0.353896 rgba(112, 112, 255, 255), stop:0.542289 rgba(218, 167, 127, 255), stop:0.79 rgba(170, 0, 255, 255), stop:0.967532 rgba(255, 158, 158, 255), stop:1 rgba(0, 0, 127, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 48pt &quot;MS UI Gothic&quot;;
</string>
    </property>
    <property name="text">
     <string>StarExpress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlanet">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>310</width>
      <height>610</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/planeta-zemlya-v2.xxl.jpg</normaloff>img/planeta-zemlya-v2.xxl.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>1000</width>
      <height>610</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMeteorite">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>311</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/original.jpeg</normaloff>img/original.jpeg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>1000</width>
      <height>700</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>311</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/437415469160280b7e9930d77d651fc7--saudi-arabia-night-skies.jpg</normaloff>img/437415469160280b7e9930d77d651fc7--saudi-arabia-night-skies.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>600</width>
      <height>650</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>730</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Планеты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>730</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Звёзды</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>730</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Метеориты</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
